src/Command/SynchronizeProductVariantsCommand.php

```diff
@@ -50,17 +50,19 @@ public function __construct(
         $this->productVariantRepository = $productVariantRepository;
     }
 
-    protected function configure()
+    protected function configure():void
```

Comment:
would expect a space after the `()`. i guess sylius has no too to check the coding style?

Reply:
`(): void` is the correct way will change it